Assets/HoloToolkit-Examples/Input/Scripts/HapticsTest.cs

```diff
@@ -26,6 +26,10 @@ void IInputHandler.OnInputDown(InputEventData eventData)
             }
         }
 
+        public void OnInputPressed(InputEventData eventData)
```

Comment:
Where is a use case for this? If you have down and up you don't need press.

Reply:
I propose removing this addition to the interface.

Reply:
I disagree.  This mirrors how Unity handles input and we should be doing that as closely as possible.

Reply:
This also opens up the Input Handler to use more generic inputs like keyboard presses, mouse clicks, etc.

Reply:
Having this addition doesn't change anything immediately and really is an addition to what's already there.

Reply:
When would I use `OnInputPressed` that I couldn't use `OnInputDown`?

Reply:
I understand it from a polling perspective, like the Unity APIs, but I'm not sure I understand it from an eventing perspective. Will the event fire constantly until the input is released? I don't see any code actually firing or using this event.

Reply:
Yeah it was never implemented in anything we're currently using, but essentially it is for polling.

Reply:
Yeah so let's not implement a polling function with our eventing story just yet :)

Reply:
Why?

Reply:
This might not be the only polling function you need. The approach needs to be more thought out. We can skip this from this release to design it better.

Reply:
I don't understand. It's matching that patter Unity uses to poll for button/key/click presses.

It's generic enough to be raised/called from any input source that implements this.

I can't think of any other way to design it better than that.

Reply:
Plus there's not really and perf overhead because of the way we've reduced allocations and reuse Events in the InputManager.